source/geod24/LocalRest.d

```diff
@@ -101,7 +104,7 @@ module geod24.LocalRest;
 static import C = geod24.concurrency;
 import geod24.Serialization;
 import std.meta : AliasSeq;
-import std.traits : Parameters, ReturnType;
+import std.traits : isDelegate, Parameters, ReturnType;
```

Comment:
Leftover ? You probably wanted `isCallable` right ?

Reply:
Haha, you got me.